### DISCUSSION TITLE
Table targetting

### DISCUSSION THREAD

**Discussion by yudhiezk** [supabase/supabase#11118] [Questions] [answered]
Hi, I'm a newbie to supabase. And I'll be making a lot of silly questions. I'm a FE developer with no BE background.

I followed a tutorial here https://supabase.com/docs/guides/getting-started/tutorials/with-nuxt-3. So it creates a table "profiles". Its super cool when I use signInUsingPassword or signInUsingOtp it directly targeting the "profiles" table. So for my first silly question. How do we set that it targets certain table?

**Comment by GaryAustin1** [comment, marked as answer]
This https://supabase.com/docs/guides/auth/managing-user-data gives a code sample that does just the addition of user data to a profile table.

You might also want to consider Supabase's Discord channel for these types of questions.
